docs/specification/1.0.md

```diff
@@ -10,12 +10,6 @@ The CDS Hooks specification describes the RESTful APIs and interactions to integ
 ### Conformance Language
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this specification are to be interpreted as described in [RFC2119](https://tools.ietf.org/html/rfc2119).
 
-### Limitations
-This specification has the following limitations.
-
-* The specification provides options for providing a CDS Service access to the EHR's FHIR resources through use of either prefetch and/or a bearer access token to be presented to the FHIR Server.  Decisions regarding which approach to take, and details regarding how the choice is implemented, are left up to the EHR vendor/provider, as architectural considerations will come into play.
```

Comment:
We state this same content in https://cds-hooks.org/specification/1.0/#providing-fhir-resources-to-a-cds-service